packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -629,6 +664,32 @@ export default class Health extends Component {
             </Card>
           </Col>
         </Row>
+        <Row className='too-many-snapshots-vms'>
+          <Col>
+            <Card>
+              <CardHeader>
+                <Icon icon='vm' /> {_('tooManySnapshots')}
+              </CardHeader>
+              <CardBlock>
+                <p>
+                  <Icon icon='info' /> <em>{_('tooManySnapshotsTip')}</em>
+                </p>
+                <NoObjects
+                  collection={
+                    props.areObjectsFetched
+                      ? this._getTooManySnapshotsVms()
+                      : null
+                  }
+                  columns={TOO_MANY_SNAPSHOT_COLUMNS}
+                  component={SortedTable}
+                  emptyMessage={_('noTooManySnapshotsObject')}
+                  shortcutsTarget='.too-many-snapshots-vms'
+                  stateUrlParam='s_too_many_snapshots_vms'
```

Comment:
What is `stateUrlParam`? is `s_too_many_snapshots_vms` a good value?

Reply:
It's a query string entry used to save the state (eg filter, page) of the sorted table in the history (via the URL).

It should work but it's a bit long.

Reply:
Ok, thanks

How about `s_many_snapshots`? Would it be better to also make the corresponding name change for all the other added variables?

Reply:
No, don't worry about it :slightly_smiling_face: